anvil-rstudio-bioconductor/scripts/set_up_package_dir.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+# This script needs to be separated from the Dockerfile because we mount persistent disks in the
+# /home/rstudio directory and when we mount the PD, everything in that directory gets wiped.
+# Therefore, we will run this script from Leonardo after the PD is mounted
+
+RSTUDIO_USER_HOME=/home/rstudio
+R_PATH=`R CMD printenv R_LIBS_USER | sed 's/~\///g'`
+BIOCONDUCTOR_VERSION=`printenv BIOCONDUCTOR_DOCKER_VERSION | sed 's/\(^[0-9].[0-9][0-9]\).*/\1/g'`
+R_PACKAGE_DIR=${RSTUDIO_USER_HOME}/${R_PATH}-${BIOCONDUCTOR_VERSION}
+
+mkdir -p ${R_PACKAGE_DIR}
```

Comment:
Does this script get run as `rstudio-user`? Probably should since it's creating directories under `/home/rstudio`

Reply:
I believe it's run as root because line 16 works just fine. maybe to the command that create dirs under `/home/rstudio` i could add `-u rstudio`

Reply:
Not sure about this, but there could be installation issues if `rstudio` user doesn't have access to write to the folder `/home/rstudio/R/....4.0-3.12/`. @gpcarr Could you test this?

Reply:
was able to confirm that this script runs as `root` user. I did test that `rstudio` user has access to write to `/home/rstudio/R/....4.0-3.12/`, there's a screenshot in the description showing packages successfully installed in this directory @nturaga. I'll do some refactoring so that the lines that write to `/home/rstudio` are run as `rstudio` user explicitly but by having the `chmod a+rwx` we are allowing this dir to be written to